HLTrigger/Configuration/python/HLT_GRun_cff.py

```diff
@@ -85882,17 +85882,15 @@
     beta = cms.double( 1.0 ),
     trackPairV0Filter = cms.PSet(  k0sMassWindow = cms.double( 0.03 ) ),
     trackSelection = cms.PSet( 
-      variableJTAParsi = cms.PSet( 
```

Comment:
AFAIK, edits in this file can not be accepted by HLT, please change HLTrigger/Configuration/python/customizeHLTforCMSSW.py  instead